nengo_loihi/builder.py

```diff
@@ -364,16 +585,18 @@ def build_ensemble(model, ens):
         bias=bias)
 
 
-def build_interencoders(model, ens):
-    """Build encoders accepting on/off interneuron input."""
+def build_inter_encoders(model, ens, kind='inter_encoders'):
```

Comment:
Why not pass in the `Interneurons` instance here rather than a string? With the class hierarchy, it should be fine to pass in any instance.

Reply:
Because the string also gets used for naming the synapse, which is then used to find the synapse by name when we're making connections.